ocaml/xapi/helpers.ml

```diff
@@ -803,6 +803,38 @@ let choose_crashdump_sr ~__context ~vm =
     | None, None ->
         raise (Api_errors.Server_error (Api_errors.vm_no_crashdump_sr, [Ref.string_of vm]))
 
+(* Returns an SR suitable for receiving patches on this host *)
```

Comment:
Perhaps it's not important, but this is not guaranteed to choose the same SR every time, even if a previously used one (with patch VDIs on it) is still usable.